cmd/bridge/main.go

```diff
@@ -237,14 +233,12 @@ func main() {
 		os.Exit(1)
 	}
 
-	capnslog.SetGlobalLogLevel(capnslog.INFO)
 	if *fLogLevel != "" {
-		llc, err := rl.ParseLogLevelConfig(*fLogLevel)
+		_, err := setLogLevel("0")
```

Comment:
If both `v` and `log-level`flags are set, won't this  override the `v` flag?

Reply:
I think we should be removing the `log-level` flag completely

Reply:
Yes, this would happen second, so it would override the -v value.  Obviously this is just in here to ensure backwards compatibility until we update. I think the right thing is to not set a default v value here.  I will remove.

Reply:
>  Obviously this is just in here to ensure backwards compatibility until we update. 
Not sure what you mean by that.
The thing is that the console binary/image/pod is only consumed by the console-operator so I'm not sure why it should be backward compatible, meaning that we should be running console by console-operator in corresponding versions.

Reply:
Console Operator will send --log-level until https://github.com/openshift/console-operator/pull/250 merges.  So we need to merge the console changes first then the Console Operator changes.  This logic was just meant to handle still receiving a --log-level value from the console operator.  Not sure if that clears anything up.

Reply:
Since we decided to keep the `--log-level` flag I would update it to:
```go
fLogLevel := fs.String("log-level", "", "DEPRECATED. Use -v flag instead.")

@zherman0 have you been testing both PRs together?

Reply:
Not recently, I have just been focused on the console changes and that it being robust no matter what the operator sends.

Reply:
Would be good to test both PRs together, before we merge any of them.